### PR TITLE
74 Make Social Media Model

### PR DESCRIPTION
**Pull request by azizbabul**
[taiga](https://tree.taiga.io/project/hmtanbir-college/us/74?kanban-status=844212)

### CODE REVIEW

File: app/models/admin/social_medium.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Admin::SocialMedium < ApplicationRecord
+  belongs_to :icon,:class_name=> "Admin::Icon", inverse_of: :admin_social_media
```
Comment (hmtanbir) [inline app/models/admin/social_medium.rb:2]: Is inverse name Okay ?

File: app/views/admin/social_media/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <div class="field">
+    <%= f.label :link %>
+    <%= f.text_field :link %>
+  </div>
+
+
+    <div class="field">
+      <%= f.label :icon_id %>
+      <%#= f.text_field :icon_id %>
+      <%= f.collection_select :icon_id,Admin::Icon.all.map,:id,:name %>
```
Comment (hmtanbir) [inline app/views/admin/social_media/_form.html.erb]: Do you check it correctly ? Is it working properly?

File: config/routes.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
   namespace :admin do
     resources :semesters, :departments, :abouts, :notice_boards,:admission_eligibilities,:tuition_fees_groups,:tuition_fees,
               :admission_fees_statuses,:scholarship_groups,:scholarships,:courses,:news,:icons,:events,:websites,
-              :quick_links,:testimonials,:google_maps,:postal_types,:postals,:videos
+              :quick_links,:testimonials,:google_maps,:postal_types,:postals,:videos,:social_media
```
Comment (hmtanbir) [inline config/routes.rb:5]: `:social_media` -- all routes are plural, so why it is singular ?

**Review by hmtanbir — CHANGES_REQUESTED** [review]

File: app/models/admin/social_medium.rb (hunk 1 above)
Comment (azizbabul) [reply app/models/admin/social_medium.rb:2]: yes data base is like that

**Review by hmtanbir — APPROVED** [review]

**Review by hmtanbir — APPROVED** [review]

**Pull request merged**